integration_tests/model_visualization_test.py

```diff
@@ -1,30 +1,47 @@
+import os
+from pathlib import Path
+
 import keras
 from keras.src.utils import plot_model
 
 
-def plot_sequential_model():
+def assert_file_exists(path):
+    assert Path(path).is_file(), "File does not exist"
+    os.remove(path)
```

Comment:
I find it useful to be able to keep the files, since I can then look at them to make sure they look as intended.

Reply:
Yes, I agree with your opinion, and removed this line. Thanks.